docs/desktop-wallet/user-guides/vote.md

```diff
@@ -1,42 +1,44 @@
 ---
-title: How to Vote or Unvote in the Desktop Wallet?
+title: How to Vote or Cancel vote in the Desktop Wallet?
 ---
 
 # How to Vote in the Desktop Wallet?
 
 ## General information
 
-You can use your SXP to vote or a delegate. Some basic voting info:
+You can use your SXP to vote for a block producer. Some basic voting info:
 
-### 1 SXP address can vote for only one delegate at a time
+### 1 SXP address can vote for up to 53 Block Producers at any time
 
-If you wish to vote for more than one delegate you can create multiple wallets, split your SXP among those wallets and vote for a different delegate with each wallet. For example, if you have 1000 SXP, you can vote for one delegate with all of your SXP in one wallet, or you can divide your SXP into two wallets of 500 SXP each and vote for two different delegates. You can create as many wallets as you want and divide your SXP among them in any proportion you choose.
+If you wish to vote for one or multiple block producers, you can simply do so from inside the Desktop Wallet. You can vote for up to 53 block producers at a time from one address. You can create as many wallets as you want and divide your SXP among them in any proportion you choose, if you want to manage your votes in a different way. Each address will be eligible to vote from 1 up to 53 block producers.
 
 ### 1 SXP equals a voting weight of 1
 
-The more SXP you hold, the more voting weight you possess. If you split your SXP into multiple wallets, your voting power divides proportionally among those wallets. For example, if you have 1,000 SXP your voting power is equal to 1,000 SXP. If you split your SXP into two wallets, each will have a voting power of 500 SXP.
+The more SXP you hold, the more votes you possess. If you split your SXP into multiple wallets, your votes divides proportionally among those wallets. For example, if you have 1,000 SXP your votes are equal to 1,000 SXP. If you split your SXP into two wallets of 500 SXP, each will have 500 votes.
```

Comment:
It's not really more "votes", it's a higher vote weight.

So something like this might be more appropriate:

> The more SXP you hold, the ~~more votes~~ higher a vote-weight you possess. If you split your SXP into multiple wallets, your ~~votes~~ vote-weight divides proportionally among those wallets. For example, if you have 1,000 SXP your ~~votes are~~ vote-weight is equal to 1,000 SXP. If you split your SXP into two wallets of 500 SXP, each will have a vote-weight of 500 ~~votes~~ .